### PR TITLE
Update windows.yml (allow ghc downgrade)

### PR DESCRIPTION
**Pull request by angerman**

### CODE REVIEW

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -31,14 +31,15 @@ jobs:
       - name: Configure Chocolatey
         run: choco source add -n mistuke -s https://www.myget.org/F/mistuke/api/v2
       - name: Install msys2
-        run: choco install -y msys2
+        run: choco install -y msys2
```
Comment (mrBliss) [inline .github/workflows/windows.yml:34]: Can you undo this change?

Comment (angerman) [reply .github/workflows/windows.yml]: 👍

**Comment by coot** [comment]
I think I found a fix in #2144. Thanks @angerman for your help.
